assets/dev/js/editor/elements/types/base/widget-base.js

```diff
@@ -0,0 +1,14 @@
+import ElementBase from './element-base';
+
+export default class WidgetBase extends ElementBase {
```

Comment:
Try to remove and use only the element base.

Reply:
I use it.